### DISCUSSION TITLE
Is it ok to use very short audio ？

### DISCUSSION THREAD

**Discussion by Michaelwhite34** [voicepaw/so-vits-svc-fork#229] [Q&A] [answered]
They come from a game and the typical length is 0.5s to 1.5s, they mostly consist a few words or no word with emotional sound like crying.

**Comment by 34j** [comment, marked as answer]
One second of the speaking part is sufficient.

**Comment by Michaelwhite34** [comment]
> One second of the speaking part is sufficient.

What if it's just sobbing or laughing ? or just one word

**Reply by 34j** [reply]
Sorry for the late response, but too much diversity in the way speakers speak can have a negative impact. You cannot know what will happen unless you try it.

**Reply by ADR3-N** [reply]
If it's just one word, I would mash it up into a big 10 second file of similar utterances, or try increasing them in intensity throughout the file.

Too much odd sounds may cause unpredictable results. You may try a file of all the related grunts.

I'm currently working on a model and have one of just the person breathing heavily (for musical effect). The result was that panting, sighing, and related utterances were better understood by the model. You really never know until you try.
